+Modules/Source.m

```diff
@@ -88,7 +88,7 @@ function blackout(obj)
         end
     end
     
-    methods(Access=private)
+    methods(Hidden)
```

Comment:
Is there a reason for this? Just curious, not protesting

Reply:
Bug with Managers. Not sure the source.